### PR TITLE
LPS-7046 Add reset button for custom screen configuration

### PR DESCRIPTION
**Pull request by Mikellorza**
This PR contains some LPD's:

1. https://liferay.atlassian.net/browse/LPD-7042
2. https://liferay.atlassian.net/browse/LPD-7045
3. https://liferay.atlassian.net/browse/LPD-7046
4. Fix about a use case of url separators validations: https://github.com/liferay-echo/liferay-portal/pull/15044/commits/0c9edb62e27833982f1514a0804b0356af13ef2b

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/display/context/FriendlyURLSeparatorCompanyConfigurationDisplayContext.java [hunk 1]
```diff
@@ -138,13 +138,24 @@ public JSONObject getErrorsJSONObject() {
 		return _jsonFactory.createJSONObject();
 	}
 
+	public String getSampleURL() {
```
Comment (jkappler) [inline modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/display/context/FriendlyURLSeparatorCompanyConfigurationDisplayContext.java:147]: @Mikellorza, I feel that the naming is a bit weird. We are getting a translated language key here right? Even though its value is a URL

Comment (Mikellorza) [reply modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/display/context/FriendlyURLSeparatorCompanyConfigurationDisplayContext.java:147]: Yes, we are getting a language key here

Comment (Mikellorza) [reply modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/display/context/FriendlyURLSeparatorCompanyConfigurationDisplayContext.java:147]: what are you thinking here?

Comment (Mikellorza) [reply modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/display/context/FriendlyURLSeparatorCompanyConfigurationDisplayContext.java]: @jkappler I added a constant with www.sitename.com

**Comment by sandrodw3** [comment]
> @sandrodw3 just one thing, when I'm navigating with keyboard and the reset button is pressed, the focus does not return to the input, since the focus stays on the button that disappears.

Thanks @veroglez! I fixed it by returning the focus to the input 😄

**Comment by veroglez** [comment]
LGTM!! :rocket:

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:report:2532896479

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
image

**Comment by Mikellorza** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by jkappler** [comment]
https://github.com/liferay-echo/liferay-portal/pull/15111

**Pull request closed without merging**